language_detector/main.py

```diff
@@ -1,9 +1,42 @@
 """This is the entry point of the program."""
-
+# -*- coding: utf-8 -*-
 from .languages import LANGUAGES
 
 
+
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    # implement your solution
+    textList = text.split()
+    returnName = ''
+    highestCount = 0
+    
+    for lang in languages:
+        langName = lang['name']
+        langWords = lang['common_words']
+        
+        count = 0  
+        for i in textList:
+            if i in langWords:
+                count += 1
```

Comment:
I like the idea of just counting how many words are similar between the two lists. However, if you were to construct a list of the shared words, you would be able to give specifically what words were the most common. This information may be valuable either to the user or in refining the constructed "common words" lists.